### DISCUSSION TITLE
Change what widget button does

### DISCUSSION THREAD

**Discussion by currall** [valinet/ExplorerPatcher#1073] [Ideas]
Could you add a way to change what the widgets button opens? I would like if it opened explorer patchers weather panel instead of Microsoft's widgets

**Comment by valinet** [comment]
At some point I will do some work in the area to clean up the functionalities.

For the moment, you can get this layout just fine:

![image](https://user-images.githubusercontent.com/6503598/158254626-44bca3da-9d77-456f-8f16-b2e1c793dc48.png)

Have the weather widget display to the left and icon only. Then, it's in the same position where the widgets button would be.

With the Windows 11 taskbar, this is not yet supported and probably won't be, at least not in the short term.

**Comment by XuanChen1234** [comment]
You can ask from @Amrsatrio
